src/EncompassRest/LoanPipeline/LoanPipelineCursor.cs

```diff
@@ -25,39 +25,41 @@ internal LoanPipelineCursor(EncompassRestClient client, string cursorId, int cou
+        public Task<LoanPipelineData> GetItemAsync(int index, IEnumerable<string> fields) => GetItemAsync(index, fields, CancellationToken.None);
 
-        public async Task<LoanPipelineData> GetItemAsync(int index, IEnumerable<string> itemFields, CancellationToken cancellationToken)
+        public async Task<LoanPipelineData> GetItemAsync(int index, IEnumerable<string> fields, CancellationToken cancellationToken)
         {
             Preconditions.GreaterThanOrEquals(index, nameof(index), 0);
             Preconditions.LessThan(index, nameof(index), Count, nameof(Count));
 
-            var data = await GetItemsInternalAsync(index, 1, itemFields, cancellationToken).ConfigureAwait(false);
+            var data = await GetItemsInternalAsync(index, 1, fields, cancellationToken).ConfigureAwait(false);
             return data[0];
         }
 
-        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count) => GetItemsAsync(start, count, null, CancellationToken.None);
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int? limit) => GetItemsAsync(start, limit, null, CancellationToken.None);
 
-        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count, IEnumerable<string> itemFields) => GetItemsAsync(start, count, itemFields, CancellationToken.None);
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int? limit, IEnumerable<string> fields) => GetItemsAsync(start, limit, fields, CancellationToken.None);
 
-        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count, IEnumerable<string> itemFields, CancellationToken cancellationToken)
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int? limit, IEnumerable<string> fields, CancellationToken cancellationToken)
         {
             Preconditions.GreaterThanOrEquals(start, nameof(start), 0);
             Preconditions.LessThan(start, nameof(start), Count, nameof(Count));
-            Preconditions.GreaterThan(count, nameof(count), 0);
-            
+            if (limit.HasValue)
+            {
+                Preconditions.GreaterThan(limit.GetValueOrDefault(), nameof(limit), 0);
+            }
 
-            return GetItemsInternalAsync(start, count, itemFields, cancellationToken);
+            return GetItemsInternalAsync(start, limit, fields, cancellationToken);
         }
 
-        private Task<List<LoanPipelineData>> GetItemsInternalAsync(int start, int count, IEnumerable<string> itemFields, CancellationToken cancellationToken) =>
-            Client.Pipeline.ViewPipelineCursorInternalAsync(CursorId, null, itemFields ?? Fields, start, count, cancellationToken, nameof(GetItemAsync), response => response.Content.ReadAsAsync<List<LoanPipelineData>>());
+        private Task<List<LoanPipelineData>> GetItemsInternalAsync(int start, int? limit, IEnumerable<string> fields, CancellationToken cancellationToken) =>
```

Comment:
Move the body of this method into `GetItemsAsync` and make the singular `GetItemAsync` call that method instead.